index.js

```diff
@@ -27,7 +27,6 @@ function whyIsNodeRunning (logger) {
   hook.disable()
   var activeResources = [...active.values()].filter(function(r) {
     if (
-      r.type === 'Timeout' &&
```

Comment:
can you check that is has a non-falsy r.resource instead, just so we avoid any regressions. otherwise 👍

Reply:
@mafintosh wouldn't that mean that async_hooks is reporting a falsy resource in that case? I don't think that's valid behavior. Maybe we should assert that `r.resource` is not falsy, wdyt?

Reply:
If the docs says that resource is never falsy then this is fine

Reply:
@mafintosh the [doc](https://nodejs.org/api/async_hooks.html#initasyncid-type-triggerasyncid-resource) for the `resource` field in `init()` uses `<Object>` as the type which doesn't include `null`. We do specify potential `null` values explicitly, like in the case for [`server.address()`](https://nodejs.org/api/net.html#serveraddress) - `Returns: <Object> | <null> | <string>`.